tutorials/misc/plot_report.py

```diff
@@ -203,6 +203,56 @@
 
 report.save('report_custom.html', overwrite=True)
 
+###############################################################################
+# Adding a stc plot to a report
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Now we see how :class:`~mne.Report` handles :class:`~mne.SourceEstimate`
+# data. The following will produce a stc plot with vertex time courses. In
+# this scenario, we also demonstrate how to use the ``screenshot`` function
+# to save the figs in a slider.
+
+fname_stc = os.path.join(path, 'MEG', 'sample', 'sample_audvis-meg')
+stc = mne.read_source_estimate(fname_stc, subject='sample')
+figs = list()
+kwargs = dict(subjects_dir=subjects_dir, initial_time=0.13,
+              clim=dict(kind='value', lims=[3, 6, 9]))
+for hemi in ('lh', 'rh'):
+    brain = stc.plot(hemi=hemi, **kwargs)
+    figs.append(brain.screenshot(time_viewer=True))
+    brain.close()
+
+# add the stc plot to the report:
+report.add_slider_to_section(figs)
+
+report.save('report_stc.html', overwrite=True)
+
+###############################################################################
+# Adding custom figs to a report
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Here we use :meth:`~mne.viz.create_3d_figure` to create a figure with custom
+# parameters. Then, we send the fig to :meth:`~mne.viz.plot_alignment`:
+
+# raw_fname = os.path.join(path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+# trans_fname = os.path.join(path, 'MEG', 'sample',
+#                            'sample_audvis_raw-trans.fif')
+# raw = mne.io.read_raw_fif(raw_fname)
+# trans = mne.read_trans(trans_fname)
+# fig = mne.viz.create_3d_figure(bgcolor=(0., 0., 0.), size=(800, 600))
+# mne.viz.plot_alignment(raw.info, trans=trans, subject='sample',
+#                        subjects_dir=subjects_dir, surfaces='head-dense',
+#                        show_axes=True, dig=True, eeg=[], meg='sensors',
+#                        coord_frame='meg', mri_fiducials='estimated', fig=fig)
+
+# # we modify the camera settings
+# mne.viz.set_3d_view(fig, 45, 90, distance=0.6, focalpoint=(0., 0., 0.))
+
+# # add the custom plot to the report:
+# report.add_figs_to_section(fig, captions='Source Alignment')
+
+# report.save('report_alignment.html', overwrite=True)
```

Comment:
should we uncomment this before merging?

Reply:
Circle does not digest this part during doc building but for me it works well locally if I execute the example.

I commented to keep it and prevent Circle from failing. What do you think?